app/forms/state_file/esign_declaration_form.rb

```diff
@@ -17,6 +17,13 @@ def save
       efile_info = StateFileEfileDeviceInfo.find_by(event_type: "submission", intake: @intake)
       efile_info&.update!(attributes_for(:state_file_efile_device_info))
 
+      unless Flipper.enabled?(:allow_duplicate_submissions)
+        if accepted_submissions_with_same_ssn(@intake).any?
+          Rails.logger.warn "#{@intake.state_code}#{@intake.id} was not submitted because there is already an accepted submission for that ssn"
```

Comment:
is this sufficient notification? would it be worth elevating this to a slack message or sentry warning or similar?